backend/backend/properties.py

```diff
@@ -100,14 +100,13 @@ def _request_property(self, table, field, attribute):
             try:
                 with conn.cursor() as cursor:
                     query_string = cursor.mogrify(
-                        'SELECT %s FROM %s',
-                        (field, table))
+                        'SELECT ({}) FROM {}'.format(field, table))
```

Comment:
The two argument version of `cursor.mogrify` that’s used below (line 109) will do more escaping of the input data, and is probably safer.

Reply:
Actually, why is this changed? Have I misunderstood something?

Reply:
I was getting errors in the SQL whenever I requested the property. The quotes were not being removed as far as I could see and it was searching the DB for '"houses"'. Which was giving the SQL errors.

Reply:
Ah, very interesting! Sorry that I didn’t test this properly when I wrote it.